### ISSUE TITLE
UnboundLocalError: local variable 'latex_datatype' referenced before assignment

### ISSUE THREAD

**Issue by ycopin** [spatialaudio/nbsphinx#50]
When using nbsphinx 0.2.8, I have an UnboundLocalError (see e.g. https://readthedocs.org/projects/spectrogrism/builds/4035544/):
```
Exception occurred:
  File "/home/docs/checkouts/readthedocs.org/user_builds/spectrogrism/envs/dev/local/lib/python2.7/site-packages/nbsphinx.py", line 804, in _get_output_type
    return html_datatype, latex_datatype
UnboundLocalError: local variable 'latex_datatype' referenced before assignment
```
I guess this relates to the fact that variable `latex_datatype` is not necessarily defined in `nbsphinx._get_output_type()` if none of the datatypes in `DISPLAY_DATA_PRIORITY_LATEX` is in `output.data`.

**Comment by mgeier** [comment]
Thanks for the detailed report. Does #51 fix it?

**Comment by mgeier** [comment]
Pinging @ycopin ...

**Comment by ycopin** [comment]
Sorry I don't have the opportunity to test the fix right now... Furthermore, I don't know if a simple warning about unsupported cell content will solve the issue: the notebooks are well rendered with nbsphinx-0.2.6 (see http://spectrogrism.readthedocs.io/en/dev/#example-notebooks).

**Comment by mgeier** [comment]
Would be great if you could test this. If you don't have time, I'll merge it anyway and wait if someone complains ...

It's not only the warning, I also added the missing assignments.

I introduced the bug in #25, which was released in nbsphinx 0.2.8, so 0.2.6 should indeed work fine.
